packages/design-system-react-native/jest.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = merge(baseConfig, {
     '!**/*.dev.{js,ts}', // Exclude .dev files
     '!**/*.assets.{js,ts}', // Exclude .assets files
     '!**/*.types.{js,ts}', // Exclude .types files
+    '!./src/components/temp-components/Blockies/Blockies.utilities.d.ts',
```

Comment:
Resolves this testing issue for new recent types file for blockies utilities 

<img width="418" alt="Screenshot 2025-05-21 at 12 51 25 PM" src="https://github.com/user-attachments/assets/3914c3fd-6605-4a0e-9ef4-1393f3d339d9" />